### DISCUSSION TITLE
File uploads not completing using Wasabi hot cloud storage

### DISCUSSION THREAD

**Discussion by needlelies** [iterate-ch/cyberduck#14956] [Q&A]
We've been having significant issues trying to upload files into our Wasabi hot cloud storage buckets through Cyberduck.

Our use is simple, we just have backup storage for our media files. 

The issue that happens is that we start uploading that completes around 99% then the upload cycle reboots and restarts without completing or just stops and we get simple error messages without any indication as to what is causing the issues. 

I've tried both having the files on my local computer or on an external drive and the error just keeps happening.
The issue is not specific to one user or system as many of us are having the same issues across multiple locations and OS systems. 

Are you aware of any others having a similar issue? We're not sure if it's a Cyberduck problem. We just tried using Wasabi Explorer and it worked fine.

**Comment by tatze96** [comment]
The described behavior usually indicates temporary connectivity issues during the transfer, causing Cyberduck to restart the current segment or the whole file.

Does this behavior occur with any file or only with specific files (file extension, file size)?

To investigate the specific cause, we would need the logging output for further investigation. 
For the logging output, please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue, and send us the log file afterward If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14956 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Comment by solidGad** [comment]
> We've been having significant issues trying to upload files into our Wasabi hot cloud storage buckets through Cyberduck.
> 
> Our use is simple, we just have backup storage for our media files.
> 
> The issue that happens is that we start uploading that completes around 99% then the upload cycle reboots and restarts without completing or just stops and we get simple error messages without any indication as to what is causing the issues.
> 
> I've tried both having the files on my local computer or on an external drive and the error just keeps happening. The issue is not specific to one user or system as many of us are having the same issues across multiple locations and OS systems.
> 
> Are you aware of any others having a similar issue? We're not sure if it's a Cyberduck problem. We just tried using Wasabi Explorer and it worked fine.

How did you make out with this? I am having a similar issue with XNS S3 provider. Larger files, gets to 99%, upload restarts and then fails. Mountain Duck is the only one of the few apps that i have had this issue. Would like to figure it out as Mountain Duck is nice to use.

**Reply by tatze96** [reply]
Can you please [enable debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log), recreate the issue, and send us the log files (*mountainduck.log*, *mountainduck-\*.log.zip*) to support@mountainduck.io afterward? You can ZIP the log files or share them through a cloud storage of your choice if they are too big for sending.

Please make sure to add the ticket reference GH-14956 for faster ticket allocation.
